opt_config.yml

```diff
@@ -0,0 +1,177 @@
+#Valid options: ['tree_parzen_estimators', 'random_search']. Mandatory field.
```

Comment:
comments (in general in the whole PR) should follow pep8 conventions:
# Add your comment (one space after hashtag for stand-alone comments, two spaces for inline ones)